tensorboard/plugins/histogram/histograms_plugin.py

```diff
@@ -84,16 +85,17 @@ def is_active(self):
             return False  # `list_plugins` as called by TB core suffices
 
         if self._multiplexer:
-            return any(self.index_impl(experiment="").values())
+            empty_context = context.RequestContext()  # not used
```

Comment:
Ditto

Reply:
[![dit]][to]

[dit]: https://user-images.githubusercontent.com/4317806/87107642-29d2bd00-c215-11ea-8d6e-2bcf4c8f6c99.png
[to]: https://i.kym-cdn.com/photos/images/original/000/303/457/f29.gif

Reply:
I suppose making me snort unwillingly during a meeting is a fair punishment for having left a silly comment not just once but twice 🤣